eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -306,6 +324,8 @@ class SqliteAuditDb(val sqlite: Connection) extends AuditDb with Logging {
       statement.setBoolean(6, m.isMultiPart)
       statement.setString(7, m.experimentName)
       statement.setBytes(8, m.recipientNodeId.value.toArray)
+      statement.setBytes(9, m.paymentHash.toArray)
+      statement.setString(10, Serialization.write(m.routingHints.map(_.value.toHex)))
```

Comment:
We usually don't put string serialization inside sqlite (we prefer deterministic binary serialization with scodec and only do json with postgres)...but in this case it's a really simple one, so maybe it doesn't hurt? I'm not sure what is best practice here, will defer to @pm47

Reply:
I would simply not implement it on sqlite.

Reply:
Reverted all the sqlite changes